lib/core/network/protocols/http.js

```diff
@@ -78,6 +85,30 @@ class HttpMessage {
     }
   }
 
+  /**
+   * Checks that an incoming HTTP message is well-formed
+   *
+   */
+  validate () {
```

Comment:
Where this method is called? I can't find the call anywhere :thinking:

Reply:
ask my brain :expressionless: